protos/flyteidl/datacatalog/datacatalog.proto

```diff
@@ -1,128 +1,223 @@
 syntax = "proto3";
 
-package pb.lyft.datacatalog;
+package datacatalog;
 
-option go_package = "datacatalog";
-option py_generic_services = true;
+import "flyteidl/core/literals.proto";
+import "google/protobuf/timestamp.proto";
 
-message Parameter {
+service DataCatalog {
```

Comment:
awesome, thank you. Maybe we should have a follow up to document better here

Reply:
Yes. The document for datacatalog is very little.